Source/GB.tnLabs.Web/API_Controllers/AzureManagementController.cs

```diff
@@ -13,9 +13,9 @@
 using System.Net;
 using System.Net.Http;
 using System.Threading.Tasks;
-using System.Web.Http;                                            
+using System.Web.Http;
 
-namespace GB.tnLabs.Web.Controllers
+namespace GB.tnLabs.Web.API_Controllers
```

Comment:
the folder name and the namespace name should be APIControllers not API_Controllers. (Bad practice)